src/components/elements/upload/fileDialog/FilePreview.tsx

```diff
@@ -179,8 +184,6 @@ const FilePreview: React.FC<{ file: FileDialogProps['file'] }> = ({ file }) => {
       </Typography>
       <Box
         sx={(theme) => ({
-          backgroundColor: theme.palette.grey[300],
-          padding: 2,
```

Comment:
Remove extra gray space that was busy and not adding anything. Before and after screenshots. This will probably be revisited more closely with design modal changes which is in the queue

<img width="772" alt="Screenshot 2024-12-30 at 11 16 11 AM" src="https://github.com/user-attachments/assets/d8f50023-c5d9-46f8-92f3-85db1cdaaf9a" />
<img width="791" alt="Screenshot 2024-12-30 at 11 17 52 AM" src="https://github.com/user-attachments/assets/930b7ef8-995a-4a99-8a91-54ec9bad4f3e" />
<img width="786" alt="Screenshot 2024-12-30 at 11 19 47 AM" src="https://github.com/user-attachments/assets/561a8c19-ab45-42f3-8af1-f843f6c250b6" />
<img width="790" alt="Screenshot 2024-12-30 at 11 19 56 AM" src="https://github.com/user-attachments/assets/a32ea6f1-e031-41ac-9159-60d219d84a21" />